index.html

```diff
@@ -3539,10 +3561,9 @@ <h3><dfn>Switch To Parent Frame</dfn></h3>
 
  <li><p><a>Handle any user prompts</a> and return its value if it is an <a>error</a>.
 
- <li><p>If the <a>current browsing context</a> is not equal to
-  the <a>current top-level browsing context</a>,
-  set the <a>current browsing context</a> to
-  the <a>parent browsing context</a> of the <a>current browsing context</a>.
+ <li><p>If the <a>current session</a>'s <a>current parent browsing
+ context</a> is not <a>null</a>, <a>set the current browsing
+ context</a> with the <a>current parent browsing context</a>.
```

Comment:
Please note that the first check for `current browsing context` of the remote steps still cause an early abort of the command, which leads to not being able to switch to the parent context, if the current one is gone.